.gitlab-ci.yml

```diff
@@ -1,27 +1,31 @@
-default:
-  interruptible: true
-
-.check_skip_rust:
+build-images:
```

Comment:
Can we move WASM building and artifacts as well to GL? SO we have all checks and tests on GH, and everything we publish we do via GL, instead of having to jump between the two.

Reply:
done, only docs publishing will stay on gh